si/orange3-text.jaml

```diff
@@ -0,0 +1,5182 @@
+    __main__: false
+    THE_GUARDIAN_API_KEY: false
+    test: false
+    refugees: false
+import_documents.py:
+    docx: false
+    odt: false
+    txt: false
+    pdf: false
+    xml: false
+    conllu: false
+    Text: Besedilo
+    name: false
+    path: false
+    ext: false
+    category: false
+    content: false
+    TextDataError: false
+    error: false
+    error_str: false
+    class `Reader`:
+        def `read`:
+            Error reading failed: false
+        def `read_file`:
+            No reader for {}: false
+        def `make_text_data`:
+            None: false
+            \s+: false
+            ' ': false
+    class `TxtReader`:
+        .txt: false
+        def `read_file`:
+            r: false
+    class `DocxReader`:
+        .docx: false
+    class `OdtReader`:
+        .odt: false
+        def `read_file`:
+            ' ': false
+    class `PdfReader`:
+        .pdf: false
+        def `read_file`:
+            rb: false
+            ' ': false
+            \x00: false
+    class `XmlReader`:
+        .xml: false
+        def `read_file`:
+            \n: false
+    class `CsvMetaReader`:
+        .csv: false
+    class `YamlMetaReader`:
+        .yaml: false
+        def `read_file`:
+            r: false
+    class `TsvMetaReader`:
+        .tsv: false
+        def `read_file`:
+            \t: false
+    def `quote_url`:
+        def `quote_byte`:
+            %{:02X}: false
+        utf-8: false
+    class `ConlluReader`:
+        Text: Besedilo
+        name: false
+        path: false
+        ext: false
+        category: false
+        doc_id: false
+        content: false
+        .conllu: false
+        def `parse_ner`:
+            misc: false
+            NER: false
+            O: false
+            lemma: false
+            ' ': false
+        def `read_file`:
+            r: false
+            utf-8: false
+            newdoc id: false
+            ' ': false
+            text: false
+            lemma: false
+            upos: false
+            ', ': false
+        def `make_text_data`:
+            None: false
+            text: false
+            \s+: false
+            ' ': false
+            newdoc id: false
+    class `ImportDocuments`:
+        Text file: Besedilna datoteka
+        ID: false
+        def `__init__`:
+            /: false
+        def `_retrieve_paths`:
+            *.{}: false
+            *.csv: false
+            *.yaml: false
+            *.yml: false
+            *.tsv: false
+        def `_create_corpus`:
+            name: false
+            path: false
+            content: false
+            utterance: false
+            category: false
+            NFC: false
+            title: false
+        def `_add_metadata`:
+            path: false
+            utterance: false
+            first: false
+        def `scan`:
+            *: false
+            .*: false
+        def `scan_url`:
+            *: false
+            .*: false
+            /: false
+language.py:
```

Comment:
Some languages can be translated, but there are some other sets of languages that can't be translated (e.g. Snowball Steemer, UDPipe Lemmatizer, Stopwords), so me and @janezd decided that we would not translate any of them to be consistent.

Reply:
Ok, that makes sense. :) I think it will be easier in the future when we finish the language transformation of the addon. Then I think all languages will be in one location.